tools/sirius_csifingerid/sirius_csifingerid.py

```diff
@@ -32,13 +32,23 @@
 parser.add_argument('--schema', default='msp')
 parser.add_argument('-a', '--adducts', action='append', nargs=1,
                     required=False, default=[], help='Adducts used')
+parser.add_argument('--credentials')
 
 args = parser.parse_args()
 print(args)
 if os.stat(args.input_pth).st_size == 0:
     print('Input file empty')
     exit()
 
+with open(args.credentials) as f:
+    credentials = json.load(f)
+
+cmd_login = "sirius login " \
+            "--user='{}' " \
+            "--password='{}'".format(credentials["username"],
+                                     credentials["password"])
```

Comment:
```suggestion
cmd_login = f"sirius login --user='{credentials["username"]}' --password='{credentials["password"]}'"
```

f-string are the new cool stuff :)